tools/topology/topology2/platform/intel/hdmi-generic.conf

```diff
@@ -64,6 +64,7 @@ Object.Pipeline {
 			index $HDMI1_DAI_PIPELINE_ID
 
 			Object.Widget.copier.1 {
+				node_type $HDA_LINK_OUTPUT_CLASS
```

Comment:
can we make the node_type a required attributes in the class definition? That would catch misses in the definition of the pipelines, no?

Reply:
Yes, that my eventual goal but I can't do that just yet. I need to clean a lot more pipelines for it